core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -0,0 +1,188 @@
+<style>
+  .oppia-sidebar-menu {
+    background: #fff;
+    height: 100%;
+    left: 0;
+    position: fixed;
+    top: 0;
+    -ms-transform: translate(-100%, 0);
+    -webkit-transform: translate3d(-100%, 0, 0);
+    transform: translate3d(-100%, 0, 0);
+    width: 270px;
+    z-index: 100;
+  }
+  .oppia-sidebar-menu-transition {
+    -webkit-transition: all 0.5s;
+    transition: all 0.5s;
+  }
+  .oppia-sidebar-menu::after {
+    background: rgba(0,0,0,0.2);
+    content: '';
+    height: 100%;
+    opacity: 1;
+    position: absolute;
+    right: 0;
+    top: 0;
+    -webkit-transition: opacity 0.5s;
+    transition: opacity 0.5s;
+    width: 100%;
+  }
+  .oppia-sidebar-menu-icon {
+    height: 22px;
+    margin: 0 16px 3px 6px;
+    vertical-align: middle;
+    width: 22px;
+  }
+  .oppia-sidebar-menu ul {
+    font-size: 16px;
+    list-style: none;
+    margin-bottom: 0;
+    padding-left: 0;
+    padding-top: 10px;
+  }
+  .oppia-sidebar-menu hr {
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+  .oppia-sidebar-menu .oppia-sidebar-section-header {
+    color: #888;
+    display: block;
+    margin-left: 10px;
+    padding: 1em 0 0 0;
+    text-decoration: none;
+  }
+  .oppia-sidebar-menu a {
+    color: #333;
+    display: block;
+    padding: 1em 0 1em 18px;
+    text-decoration: none;
+  }
+  .oppia-sidebar-menu a:hover,
+  .oppia-sidebar-menu a:focus {
+    background: rgb(243, 248, 255);
+  }
+  .oppia-sidebar-menu li.active a {
+    background: rgba(243, 248, 255, 0.5);
+  }
+  .oppia-sidebar-logo-container {
+    color: #333;
+    margin: 0 auto;
+    width: 120px;
+  }
+
+  .oppia-sidebar-header {
+    background: #eee;
+    border-bottom: 1px solid #ccc;
+    height: 56px;
+    padding-top: 6px;
+  }
+  .oppia-sidebar-menu-icon.fa {
+    font-size: 24px;
+  }
+  .classroom-submenu {
+    padding: 0 0 1em 18px;
+  }
+  .oppia-sidebar-menu-open {
+  box-shadow: 1px 0 3px rgba(0,0,0,0.12), 1px 0 2px rgba(0,0,0,0.24);
+  height: 110vh;
+  overflow-y: scroll;
+  -ms-transform: translate(0, 0);
+  -webkit-transform: translate3d(0, 0, 0);
+  transform: translate3d(0, 0, 0);
+  visibility: visible;
+  }
+  .oppia-sidebar-menu-open::after {
+    height: 0;
+    opacity: 0;
+    -webkit-transition: opacity 0.5s, width 0.1s 0.5s, height 0.1s 0.5s;
+    transition: opacity 0.5s, width 0.1s 0.5s, height 0.1s 0.5s;
+    width: 0;
+  }
+</style>
```

Comment:
Why move this css from oppia.css to component?
Because component's css was taking priority over the global css due to view encapsulation (unlike ajs).

Reply:
this was causing issues showing sidebar